test/bpf/bpf.vcxproj

```diff
@@ -34,14 +34,16 @@
   <ItemDefinitionGroup>
     <PreBuildEvent>
       <Command>
-       set PATH=%PATH%;$(EbpfPackagePath)build\native\bin
+        xcopy $(EbpfPackagePath)build\native\bin\EbpfApi.dll $(SolutionDir)artifacts\bin\$(Platform)_$(Configuration) /Y
```

Comment:
why are we copying the DLL into the bin directory? i see we're removing it later on. is the issue that the path may already have a different eBPF version on it? if so, perhaps we should fix the path line above to place the local eBPF package first, i.e. prepend it to PATH.

Reply:
export_program_info.exe and xdp_bpfexport.exe both require ebpfapi.dll. The earlier code was setting `$(EbpfPackagePath)build\native\bin` in PATH. But that was not working for some reason on my local build, and build was failing as bpfexport exe could not be executed. Hence changed the logic to copy ebpfapi.dll to the same location.

Reason for deletion:
If ebpfapi.dll is present in the local folder, test exe links loads the local ebpfapi.dll (which previously came from NuGet package). ebpfapi.dll in NuGet package is incompatible with NativeOnly ebpf driver. There is an issue tracking this: https://github.com/microsoft/ebpf-for-windows/issues/3126. Once the issue in eBPF repo is fixed, we dont need to delete the dll anymore.

Reply:
Hmmm, that's very strange. And yeah we should remove it from `bin` because XDP is meant to work with any backward-compatible version of eBPF, not just the version it was built with. Anyways, LGTM.